app/forms/DescribeYourItemFormProvider.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import javax.inject.Inject
+
+import forms.mappings.Mappings
+import play.api.data.Form
+import play.api.data.Forms._
+import models.DescribeYourItem
+
+class DescribeYourItemFormProvider @Inject() extends Mappings {
+
+   def apply(): Form[DescribeYourItem] = Form(
+     mapping(
+      "field1" -> text("describeYourItem.error.field1.required")
```

Comment:
Here the field names are not appropriate in my opinion

Reply:
See last comment reply.